web/src/helpers/consts.ts

```diff
@@ -1,5 +1,6 @@
 // UNKNOWN_ID is the symbol for unknown id
 export const UNKNOWN_ID = -1;
+export const UNKNOWN_USERNAME = "-";
```

Comment:
How about using empty string instead of `-`?
```suggestion
export const UNKNOWN_USERNAME = "-";
```

Reply:
by analogy with UNKNOWN_ID, UNKNOWN_USERNAME is substituted into the query and returns an empty result, since the user "-" was not found. I did not alter the logic of the project much

Reply:
Let's use the empty string, because `Boolean("") === false` but `Boolean("-") !== false`.

Reply:
ok, fixed to empty string